contracts/evoting/types/http.go

```diff
@@ -11,6 +11,13 @@ type LoginResponse struct {
 	UserID string
 	Token  string
```

Comment:
Now that we use signed requests, we no longer need to use tokens and can remove them from all requests in this file
```suggestion

```